perfect_num.py

```diff
@@ -0,0 +1,27 @@
+"""
+This Program prints all the perfect numbers from 0 to infinity.
```

Comment:
Yarden, you need to document inside the functions and according to the conventions.